### PR TITLE
Version upgrade after release of core-amqp 2.0.0

### PR DESCRIPTION
**Pull request by hemanttanwar**
Upgrading azure-core-amqp version after GA release of version 2.0.0

GA 2.0.0 was released from a release branch. (https://dev.azure.com/azure-sdk/internal/_build/results?buildId=636445&view=results)

### CODE REVIEW

File: eng/versioning/version_client.txt [hunk 1]
```diff
@@ -177,7 +177,6 @@ com.microsoft:microsoft-opentelemetry-exporter-azuremonitor;1.0.0-beta.1;1.0.0-b
 # unreleased_<groupId>:<artifactId>;dependency-version
 # note: The unreleased dependencies will not be manipulated with the automatic PR creation code.
 unreleased_com.azure:azure-core;1.11.0
```
Comment (alzimmermsft) [inline eng/versioning/version_client.txt:179]: Mind getting rid of this? We've already released azure-core 1.11.0

**Review by alzimmermsft — APPROVED** [review]

**Review by ChenTanyi — APPROVED** [review]

**Pull request merged**